examples/counter_in/read_freq.py

```diff
@@ -0,0 +1,27 @@
+"""Example of counter input frequency acquisition.
+
+This example demonstrates how to measure frequency using one
+counter on a Counter Input Channel. The Edge, Minimum Value and
+Maximum Value are all configurable.
+"""
+
+import nidaqmx
+from nidaqmx.constants import Edge, FrequencyUnits
+
+
+with nidaqmx.Task() as task:
+    channel = task.ci_channels.add_ci_freq_chan(
+        "Dev1/ctr0",
+        min_val=100.0,
+        max_val=1000.0,
+        units=FrequencyUnits.HZ,
+        edge=Edge.RISING,
+    )
+    channel.ci_freq_term = "/Dev1/PFI8"
```

Comment:
Every counter has a default PFI term, so we don't *have* to set these.

Whatever we choose, we should be consistent between CI and CO